editor/animation_track_editor.cpp

```diff
@@ -4889,6 +4891,7 @@ void AnimationTrackEditor::_notification(int p_what) {
 			bezier_edit_icon->set_icon(get_editor_theme_icon(SNAME("EditBezier")));
 			snap->set_icon(get_editor_theme_icon(SNAME("Snap")));
 			view_group->set_icon(get_editor_theme_icon(view_group->is_pressed() ? SNAME("AnimationTrackList") : SNAME("AnimationTrackGroup")));
+			function_name_toggler->set_icon(get_editor_theme_icon(SNAME("FontItem")));
```

Comment:
```suggestion
			function_name_toggler->set_icon(get_editor_theme_icon(SNAME("MemberMethod")));
```
MemberMethod icon is better IMO.